pagopa/pn-frontend-e2e
# PN-10200-e-2-e-refactor-test-su-i-download-degli-allegati-di-pa

## Short description
There are some test failing related to the download method.

## List of changes proposed in this pull request
- updated `tokenLogin.yaml` with description of each token
- adjusted download methods
- adjusted `waitTime` and cleaned unused code
- updated `WebElements
`
## How to test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_MittenteDownloadAttestazioneMancatoRicapDigitale` and verify if the test works fine.
